tensor2tensor/data_generators/text_encoder.py

```diff
@@ -36,6 +37,10 @@
 EOS = '<EOS>'
 RESERVED_TOKENS = [PAD, EOS]
 
+# Character that will be used instead of those with
+# ordinal numbers > 255
+UNK_CHAR = '×' # ordinal 215
```

Comment:
ok, seems reasonable. please add a `tf.logging.warn` (or a log_every_n) when that happens. user needs to know that we don't really support unicode with that encoder. 

we may want to consider moving entirely to bytes in the future and not dealing with unicode at all. but it requires more thought...